athena-postgresql/etc/test-config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "athena_work_group" : "FederationIntegrationTests", /* The Athena Workgroup used for running integration tests (default: FederationIntegrationTests) */
+  "secrets_manager_secret" : "<secret name>",         /* Secret name used to retrieve user credentials from SecretsManager. */
+  "environment_vars" : {                  /* Parameters used by the connector's internal logic */
+    "spill_bucket" : "<spill bucket>",    /* The S3 bucket used for spilling excess data */
+    "spill_prefix" : "athena-spill",      /* The prefix within the S3 spill bucket (default: athena-spill) */
+    "disable_spill_encryption" : "false"  /* If set to true encryption for spilled data is disabled (default: false) */
+  },
+  "vpc_configuration" : {                 /* VPC configuration for DB instances within a VPC */
+    "vpc_id": "<VPC Id>",                 /* The VPC Id (e.g. vpc-xxx) */
+    "security_group_id": "<SG Id>",       /* The Security Group Id (e.g. sg-xxx) */
+    "subnet_ids": ["<Subnet 1>", "<Subnet 2>"],     /* A list consisting of at least one Subnet Id (e.g. subnet-xxxx) */
+    "availability_zones": ["<Zone 1>", "<Zone 2>"]  /* A list consisting of at least one AZ (e.g. us-east-1a) */
+  },
+  "user_settings" : {                     /* User customizable settings */
+    "redshift_db_name": "<DB Name>",      /* Name of the Db used by the Redshift integration tests */
+    "redshift_db_port": "<DB Port>",      /* Port number associated with the Redshift cluster endpoint */
+    "redshift_table_movies": "<Table Name>",    /* Redshift table name */
+    "redshift_table_bday": "<Table Name>",      /* Redshift table name */
+    "postgres_db_name": "<DB Name>",      /* Name of the Db used by the PostGreSql integration tests */
```

Comment:
nit lowercase the G in PostGres

Reply:
Actually this was how it appears across the codebase, so this is consistent.

Reply:
e.g. https://github.com/awslabs/aws-athena-query-federation/blob/master/athena-jdbc/src/main/java/com/amazonaws/athena/connectors/jdbc/postgresql/PostGreSqlCompositeHandler.java#L31

Reply:
sounds good, ok with it since its for consistency